src/styles/blocks/place.scss

```diff
@@ -0,0 +1,49 @@
+.place {
+  &__photo-container {
+    position: relative;
+    width: 100%;
+    max-height: 256px;
```

Comment:
According to the design, on small screens, these pictures should become smaller (height 131px). You have a height of 256px for all widths.

Reply:
They become smaller on small screens, this max-height doesn`t let them become higher than 256px, because on small screens it looks bad

Reply:
Really. My bad. Haven't noticed. Sorry